src/kernel/context.h

```diff
@@ -18,24 +18,9 @@ namespace kernel {
 //! State stored directly in this struct should be simple. More complex state
 //! should be stored to std::unique_ptr members pointing to opaque types.
 struct Context {
-    //! Interrupt object that can be used to stop long-running kernel operations.
-    util::SignalInterrupt interrupt;
```

Comment:
Just noticed while reviewing #29252...

This was removed, but the `#include <util/signalinterrupt.h>` include wasn't. I can open a quick cleanup PR, or would you like to @ryanofsky ?

(Looks like `<memory>` can go too.)

Reply:
Good catch. I don't think I would open a new PR that just removing an unnecessary #include without having a larger goal, but I wouldn't object to reviewing one. Maybe the include fixes could be part of #29252, too.